integration_tests/commands/async/hscan_test.go

```diff
@@ -0,0 +1,86 @@
+package async
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+var THREE int64 = 3
+var FOUR int64 = 4
+
+func TestHSCAN(t *testing.T) {
```

Comment:
Please add a test case where a new cursor value is returned, then use that cursor value to get the remaining elements.

Reply:
have added the test case